homeassistant/components/aws/__init__.py

```diff
@@ -109,14 +111,17 @@
             if isinstance(result, Exception):
                 _LOGGER.error(
                     "Validating credential [%s] failed: %s",
-                    name, result, exc_info=result
+                    name,
+                    result,
+                    exc_info=result,
                 )
                 validation = False
             else:
                 hass.data[DATA_SESSIONS][name] = result
 
-    # No entry support for notify component yet
-    for notify_config in conf.get(CONF_NOTIFY, []):
+    # set up notify platform, no entry support for notify component yet,
+    # have to use discovery to load platform
+    for notify_config in conf[CONF_NOTIFY]:
         discovery.load_platform(hass, "notify", DOMAIN, notify_config, config)
```

Comment:
I missed it last round. We should use the async version `async_load_platform` here.

Reply:
I copied that code from 
https://github.com/home-assistant/home-assistant/blob/a59487a438f653063c254a7133dc6c2297877e3c/homeassistant/components/ios/__init__.py#L206

@robbiet480, you may want to change ios component as well.